inc/functions.php

```diff
@@ -178,6 +178,55 @@ function rwmb_the_value( $field_id, $args = array(), $post_id = null, $echo = tr
 	}
 } // End if().
 
+if ( ! function_exists('rwmb_get_object_fields') ) {
+	/**
+	 * Get defined meta fields for object
+	 *
+	 * @param int|string	$object_id Object id. ID of post, term.
+	 * @param string		$object_type Object type. Use post, term.
+	 *
+	 * @return array
+	 */
+	function rwmb_get_object_fields( $object_id, $meta_type = 'post' ) {
```

Comment:
Should the 2nd parameter named `$object_type`? From the function docs, it's `$object_type`.

Reply:
It is first commit. I though you can tell me how to get user defined fields as i need to know...

Reply:
@rilwis i see that and i very appreciate it that you find inconsistence.
But like in my issue which points to this PR (https://github.com/wpmetabox/meta-box/issues/1278) i want to made something like ACF helper function https://github.com/AdvancedCustomFields/acf/blob/c8e181c848009aa9f6a3919ee19981fb2774f634/includes/api/api-template.php#L207
And to do something like that i need to get all registered fields for User meta. There is plugin MB User meta, but that plugins doesn't have metabox registry class. So there is no other way how to get user meta registered fields? Then my function could looking into `post`, `term` or `user` meta fields.

Reply:
Hey, what I ask is just the simple thing: you're using **one** variable with **two** names. That's all.

Regarding the user fields, it's in the **meta box registry**. That registry is used for all object types. You can `var_dump` to see its content.

Reply:
@rilwis need to check it again, because i have all MB extensions and we use User Meta. But `rwmb_get_registry('meta_box')` does not return it, what is weird.